src/Tests/CollectionRuleActions.UnitTests/CollectLogsActionTests.cs

```diff
@@ -43,7 +51,7 @@ public CollectLogsActionTests(ITestOutputHelper outputHelper)
         /// at the log level specified in the request body.
         /// </summary>
         /// 
-        [Theory]
+        [Theory(Skip = "Haven't integrated PipeEgress into egress extensibility model yet")]
```

Comment:
@jander-msft These tests were failing due to the addition of the `PipeEgressProvider` - my thinking is that it's probably best to temporarily disable them since it would be hard to review the changes required to support it as part of the merge PR, but I'll leave that up to you.

Reply:
Makes sense. One way to re-enable this (not necessarily now) is to make this egress provider an extension and have it communicate back over named pipe or Unix domain socket.